pontoon/contributors/templates/contributors/settings.html

```diff
@@ -160,4 +172,5 @@ <h3><span class="small stress">Preferred locales to get suggestions from</span><
 
 {% block extend_js %}
   {% javascript 'settings' %}
+  {% javascript 'theme-switcher' %}
```

Comment:
The theme-switcher.js is already loaded as part of the base template and we also don't have a 'theme-switcher' defined in PIPELINE_JS, so let's revert this line.